README.md

```diff
@@ -91,7 +91,7 @@ cd ./my-project
 
 This will install an empty skeleton application, 
 but we're also offering a demo package for your convenience - of course also with 3 commands 💪
-[Click here for more installation options and a detailed guide](https://pimcore.com/docs/pimcore/current/Getting_Started/Installation.html)
+[Click here for more installation options and a detailed guide](https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html)
```

Comment:
I don't think this is correct (for the `11.x` branch). The current (`10.x`) docs are under the `Development_Documentation` path, but as far as I know the new ones we're moved to the path without it (but they're not yet live, I think).

Reply:
makes sense, that may be the reason why those are not public.